terraform/050-pw-manager/vars.tf

```diff
@@ -140,6 +140,11 @@ variable "email_signature" {
   type        = string
 }
 
+variable "extra_hosts" {
```

Comment:
If no `extra_hosts` is provided, what will happen? Wondering if we need a default value to prevent its absence from breaking the task definition JSON.

Reply:
Great observation. I'll add `null` as the default. Thanks.